java/openjdk-java/portfile

```diff
@@ -0,0 +1,380 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+name                openjdk-java
+version             17
+categories          java
+platforms           darwin
+description         an Open Source, Open Innovation, Open Standards implementation of the Java Platform, Standard Edition, and related projects.
+homepage            openjdk.java.net
```

Comment:
That's not a URL.

Reply:
openjdk.java.net is a url. ( click here - openjdk.java.net)

Reply:
Url has been changed (see:- https://openjdk.java.net)

Reply:
Fixed url